src/scss/components/loader/_hx-busy.scss

```diff
@@ -11,7 +11,6 @@
 // 1: explicitly set transform-box for IE/Edge compatibility
 // 2: explicitly set transform-origin with full X/Y/Z values for IE/Edge compatibility
 hx-busy {
-  animation-play-state: running;
```

Comment:
`animation-play-state: running;` is the default value and is unnecessary, given that the `animation` shorthand on the next line would technically override the value of `animation-play-state` anyway.